tools/bump-version.py

```
@@ -67,50 +59,66 @@ def hex_lit(version: Any) -> str:
-    elif parts[-1] == "legacy":
-        bump_header(
-            project / "firmware" / "version.h",
-            VERSION_MAJOR=major,
-            VERSION_MINOR=minor,
-            VERSION_PATCH=patch,
-        )
-    elif parts[-1] == "python":
-        bump_python(project / "python", f"{major}.{minor}.{patch}")
+        if project.match("core/embed/projects/firmware"):
+            # also bump language JSONs
+            subprocess.check_call(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)